package.json

```diff
@@ -84,6 +84,7 @@
     "react-redux": "4.4.5",
     "rebass": "0.3.0",
     "redux": "3.5.2",
-    "redux-thunk": "2.1.0"
+    "redux-thunk": "2.1.0",
+    "reselect": "^3.0.1"
```

Comment:
Also, this `package.json` is kind of a mess in general, we'll talk about that soon...